cpu/stm32/stmclk/stmclk_l4wx.c

```diff
@@ -593,6 +606,13 @@ void stmclk_init_sysclk(void)
         stmclk_dbp_unlock();
         RCC->BDCR |= RCC_BDCR_LSEON;
         while (!(RCC->BDCR & RCC_BDCR_LSERDY)) {}
+
+    /* Set LSE system clock enable bit. This is required if LSE is to be used by
+       USARTx, LPUARTx, LPTIMx, TIMx, RNG, system LSCO, MCO, MSI PLL mode */
+#if defined(CPU_FAM_STM32WL)
+        RCC->BDCR |= RCC_BDCR_LSESYSEN;
+        while (!(RCC->BDCR & RCC_BDCR_LSESYSRDY)) {}
+#endif
```

Comment:
This is already done in cpu/stm32/stmclk/stmclk_common.c, is that normal?

Reply:
`stmclk_enable_lfclk()` in `cpu/stm32/stmclk/stmclk_common.c` has other dependancies. In two of the cases LSE is expected to be enabled but with `STM32WL` there is an additional bit .